vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/AbsMessageItem.kt

```diff
@@ -163,7 +164,9 @@ abstract class AbsMessageItem<H : AbsMessageItem.Holder> : BaseEventItem<H>() {
 
     protected fun renderSendState(root: View, textView: TextView?) {
         root.isClickable = informationData.sendState.isSent()
-        textView?.setTextColor(colorProvider.getMessageTextColor(informationData.sendState))
+        val state = SendState.UNSENT.takeIf { informationData.hasPendingEdits }
```

Comment:
I think a basic If else would be more readable

Reply:
done